src/main/java/com/dungeoncrawler/Javiarenka/equipment/StartingArmor.java

```diff
@@ -7,8 +7,8 @@
 import java.util.List;
 
 public final class StartingArmor {
-    public static final Armor CLOTH_ARMOR = new Armor("Cloth Armor", List.of(HeroClass.WIZARD, HeroClass.ROGUE, HeroClass.HEALER), 5, 10, 4);
-    public static final Armor LEATHER_ARMOR = new Armor("Leather Armor", List.of(HeroClass.ARCHER, HeroClass.ROGUE, HeroClass.WARRIOR), 8, 20, 10);
-    public static final Armor RUSTED_CHAIN_ARMOR = new Armor("Rusted Chain Armor", List.of(HeroClass.KNIGHT, HeroClass.WARRIOR, HeroClass.ARCHER), 12, 30, 15);
+    public static final Armor CLOTH_ARMOR = new Armor("Cloth Armor", List.of(HeroClass.WIZARD, HeroClass.ROGUE, HeroClass.HEALER), 5, 10, 4, 90);
```

Comment:
Super, że pomyślałaś od razu o uzupełnieniu innych klas - taki ogólny pogląd jest mega ważny w programowaniu

Reply:
Teraz zauważyłam, że te wartości, które wpisałam są odwrotnie do tego co chciałam, czyli tam gdzie wpisałam 90, powinno być 10, bo ta wartość to redukcja szansy na trafienie, a nie szansa na trafienie. Poprawić to razem z metodą attack?